tools/rest_api.rb

```diff
@@ -1,293 +1,8 @@
-#!/usr/bin/env ruby
-#
-# Helper Script to access the ManageIQ REST API
-#
-# Makes use of the Trollop and Faraday Client Gems
-#   gem install trollop
-#   gem install faraday
-#   gem install faraday_middleware
-#
-# Run without options to see the usage.
-#
-# Globals accessible by loaded test scripts
-#
+#! /usr/bin/env ruby
+require 'bundler/setup'
+require 'pathname'
 
-require 'uri'
-require 'json'
-require 'trollop'
-require 'faraday'
-require 'faraday_middleware'
+# TODO: Once we rename the rest_api.rb in the manageiq-api gem and fix the "location" below,
+# we should transition to having users using the exe provided by the gem and remove this "wrapper" script.
+require Pathname.new(Gem::Specification.find_by_name('manageiq-api').gem_dir).join('exe', 'rest_api')
```

Comment:
@jrafanie Thoughts on using the following instead?

```ruby
Bundler.locked_gems.specs.select {|g| g.name == "manageiq-api"}.first.gem_dir
```

They results seem to return the same thing, just curious if we would prefer to go through bundler instead of potentially getting something incorrect from rubygems.